internal/factory/container/container_test.go

```diff
@@ -98,10 +99,30 @@ var _ = t.Describe("Container", func() {
 			mountPoint := "test"
 			configStopSignal := "test"
 
-			sb, err := sandbox.New("sandboxID", "", "", "", "test",
-				make(map[string]string), make(map[string]string), "", "",
-				&types.PodSandboxMetadata{}, "", "", false, "", "", "",
-				[]*hostport.PortMapping{}, false, currentTime, "", nil, nil)
+			sbuilder := sandbox.NewBuilder()
+
+			sbuilder.SetID("sandboxID")
+			sbuilder.SetName("")
+			sbuilder.SetNamespace("")
+			sbuilder.SetKubeName("")
+			sbuilder.SetLogDir("test")
+			sbuilder.SetCriSandbox(sbuilder.ID(), currentTime, make(map[string]string), make(map[string]string), &types.PodSandboxMetadata{})
+			sbuilder.SetShmPath("")
+			sbuilder.SetCgroupParent("")
+			sbuilder.SetPrivileged(false)
```

Comment:
similar here, only call if we're setting non default

Reply:
these notes apply to all the _test.go functions, but not for the pieces that run in production

Reply:
Resolved